auslib/test/admin/views/test_rules.py

```diff
@@ -316,13 +340,14 @@ def testPostRuleOutdatedData(self):
         self.assertEquals(r[0]['data_version'], 2)
 
         # OutdatedDataVersion Request
-        ret2 = self._put('/rules/1', data=dict(backgroundRate=71, mapping='d', priority=73, data_version=1, product='Firefox', channel='nightly'))
+        ret2 = self._put('/rules/1', data=dict(backgroundRate=71, update_type='minor', mapping='d', priority=73,
+                                               data_version=1, product='Firefox', channel='nightly'))
         self.assertEquals(ret2.status_code, 400, "Status Code: %d, Data: %s" % (ret2.status_code, ret2.data))
 
     def testPostByAlias(self):
         # Make some changes to a rule
         ret = self._post('/rules/frodo', data=dict(backgroundRate=71, mapping='d', priority=73, data_version=1,
-                                                   product='Firefox', channel='nightly'))
+                                                   product='Firefox', channel='nightly', update_type="minor"))
```

Comment:
Making update_type a required argument for rules that already exist is going to be problematic. We've got tools and scripts that make changes to existing Rules that will break because of this. Is it possible to make update_type required for new rules, but optional for existing ones? Maybe by allowing type to be string or null like you've done or other fields?